addons/loyalty/views/loyalty_program_views.xml

```diff
@@ -20,81 +20,125 @@
                                 <span class="o_stat_value">
                                     <field name="coupon_count"/>
                                 </span>
-                                <span class="o_stat_text" attrs="{'invisible': [('program_type', '!=', 'coupons')]}">Coupons</span>
+                                <span class="o_stat_text" attrs="{'invisible': ['&amp;', ('program_type', '!=', 'coupons'), ('program_type', '!=', 'next_order_coupons')]}">Coupons</span>
```

Comment:
AND is the default to the &amp here is not necessary, it would be easier to read then
```suggestion
                                <span class="o_stat_text" attrs="{'invisible': [('program_type', '!=', 'coupons'), ('program_type', '!=', 'next_order_coupons')]}">Coupons</span>
```